app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -1,23 +1,33 @@
 ---
 title: "Get an international relocation payment"
+subcategory: If you're already a teacher
 description: |-
-  Non-UK teachers and trainees in physics and languages could be eligible for financial help in the form of an international relocation payment from the UK government.
+  Non-UK teachers in physics and languages could be eligible for financial help in the form of an international relocation payment from the UK government.
 related_content:
     Train to teach in England as a non-UK citizen : "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
     Teach in England if you trained outside the UK: "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
     Apply for your visa to teach in England: "/non-uk-teachers/visas-for-non-uk-teachers"
     Apply for your visa to train to teach in England: "/non-uk-teachers/visas-for-non-uk-trainees"
 promo_content:
+navigation: 20.50
+navigation_title: Get an international relocation payment
+navigation_description: Teachers of languages or physics could be eligible for financial help from the UK government worth £10,000.
 date: "2021-05-27"
 image: "static/images/content/hero-images/0002.jpg"
 backlink: "../../"
 talk_to_us: false
 inset_text:
   applications-open:
     text: |-
-      <p>Applications for the international relocation payment (IRP) for teachers and salaried trainees are now closed. They will reopen between 2 April and 31 May 2024. 
+      <p>Applications for the <a href="https://get-a-teacher-relocation-payment.education.gov.uk/closed">international relocation payment (IRP)</a> are open between 2 April and 31 May 2024.</p>
```

Comment:
Are these applications now only open for teachers? Think it would be good to make that clear here.